modules/elasticsearch-impl/src/main/java/org/opencastproject/elasticsearch/impl/AbstractElasticsearchIndex.java

```diff
@@ -626,7 +626,7 @@ protected String getSubIndexIdentifier(String type) {
     return this.indexIdentifier + "_" + type;
   }
 
-  protected RestHighLevelClient getClient() {
+  public RestHighLevelClient getClient() {
```

Comment:
`SearchServiceIndex` extends `AbstractIndexProducer`, but not `AbstractElasticsearchIndex`, so this needs to be public. `AIP` doesn't extend anything (merely implements), but I didn't notice the two Index related classes were unrelated until final notes were being made. Could investigate having `AIP` extend `AEI`, and then this could remain protected, but that's future work imo.